spring/spring-boot-3/pom.xml

```diff
@@ -21,7 +21,7 @@
         <maven.compiler.target>17</maven.compiler.target>
         <maven.compiler.release>17</maven.compiler.release>
         <infinispan.cluster.stack>tcp</infinispan.cluster.stack>
-        <version.micrometer>1.10.3</version.micrometer>
+        <version.micrometer>1.13.9</version.micrometer>
```

Comment:
Updating the micrometer version in a patch release doesn't seem right. Is it required?

Reply:
yes

Reply:
this is only for testing